doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -314,6 +343,36 @@ messages and additional metadata helpful for debugging.
+<!-- to log nothing; a value of 0.8 means to log approximately 80% of -->
+<!-- statement executions. If `statement_logging_sample_rate` is higher -->
+<!-- than `statement_logging_max_sample_rate` (which is set by Materialize -->
+<!-- and cannot be changed by users), the latter is used instead. -->
+
+<!-- | Field                   | Type                         | Meaning                                                                                                                                                                                                                        | -->
+<!-- |-------------------------|------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| -->
+<!-- | `id`                    | [`uuid`]                     | The ID of the execution event.                                                                                                                                                                                                 | -->
+<!-- | `prepared_statement_id` | [`uuid`]                     | The ID of the prepared statement being executed. Corresponds to [`mz_prepared_statement_history.id`](#mz_prepared_statement_history).                                                                                          | -->
+<!-- | `sample_rate`           | [`double precision`]         | The sampling rate at the time the execution began.                                                                                                                                                                             | -->
+<!-- | `params`                | [`text list`]                | The values of the prepared statement's parameters.                                                                                                                                                                             | -->
+<!-- | `began_at`              | [`timestamp with time zone`] | The time at which execution began.                                                                                                                                                                                             | -->
+<!-- | `finished_at`           | [`timestamp with time zone`] | The time at which execution ended.                                                                                                                                                                                             | -->
+<!-- | `was_successful`        | [`bool`]                     | Whether the statement executed successfully. Mutually exclusive with `was_canceled` and `was_aborted`.                                                                                                                         | -->
```

Comment:
Do we also need an error state?

Reply:
That is implicit in whether the error message is `NULL` or not. I agree that that's a bit confusing, so I'll add an `e` state when I switch this to using a char-based enum.